frontend/src/metabase/dashboard/selectors.ts

```diff
@@ -185,7 +186,16 @@ export const getLoadingDashCards = (state: State) =>
 
 export const getDashboardById = (state: State, dashboardId: DashboardId) => {
   const dashboards = getDashboards(state);
-  return dashboards[dashboardId];
+
+  const isEntityId = isBaseEntityID(dashboardId);
```

Comment:
can't just simply check if `typeof dashboardId === 'string'` because the id could be in JWT form (static embed)